src/routes/webhooks/paddle.ts

```diff
@@ -203,10 +225,16 @@ export const paddle = async (fastify: FastifyInstance): Promise<void> => {
                 );
                 break;
               case EventName.SubscriptionUpdated:
-                await logPaddleAnalyticsEvent(
-                  eventData,
-                  AnalyticsEventName.ChangeBillingCycle,
-                );
+                if (!(await planChanged(eventData))) {
```

Comment:
For readability, would this be better?
```suggestion
				const didPlanChange = await planChanged(eventData);
                if (!didPlanChange) {
```

Reply:
Agreed :D